sdk/src/test/scala/io/horizen/account/state/AccountStateTest.scala

```diff
@@ -186,6 +176,40 @@ class AccountStateTest
     assertEquals(state.isSwitchingConsensusEpoch(intToConsensusEpochNumber(currentEpochNumber.get)), true)
   }
 
+  @Test
+  def testSwitchingConsensusEpochsWith2Forks(): Unit = {
+    ConsensusParamsUtil.setConsensusParamsForkActivation(Seq(
+      ConsensusParamsForkInfo(0, ConsensusParamsFork.DefaultConsensusParamsFork),
+      ConsensusParamsForkInfo(0, ConsensusParamsFork(100,10)),
```

Comment:
Changed to activation epoch 20 and adjusted the code for it
Switched to using `TimeToEpochUtils` to calculate block timestamp instead of doing it manually